docker-compose/adaptive-sampling/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+services:
+  jaeger:
+    image: jaegertracing/jaeger:${JAEGER_IMAGE_TAG:-latest}
+    volumes:
+      - "./jaeger-v2-config.yml:/etc/jaeger/config.yml"
+    command: ["--config", "/etc/jaeger/config.yml"]
+    ports:
+      - "16686:16686"
+      - "5778:5778"
```

Comment:
How about 4318 for receiving spans?

Reply:
@yurishkuro why do we need to add 4318 if its not being accessed outside of docker?

Reply:
tracegen submits to this port using jaeger:4318 address. If you don't expose the port, are you sure it's reachable from another container?

Reply:
It looks like it was working fine without it. I added it in anyway though. I wonder if its because theyre in the same network?